device.py

```diff
@@ -10,18 +11,28 @@ class Device():
     name = ""
     room = ""
     state = 0
+    brightness = 0
+    hasBrightness = False
 
-    def __init__(self, id, name, room, state):
+    def __init__(self, id, name, room, state, hasBrightness, brightness):
         self.id = id
         self.name = name
         self.room = room
         self.state = state
+        self.hasBrightness = hasBrightness
```

Comment:
Sure; I've been writing a lot of Java lately so I've gotten my mind out of the duck-typing game. I will make the adjustment.